data/hai/hai reveal 4 middle.txt

```diff
@@ -1413,13 +1413,133 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			choice
+				`	"It was a constructive discussion."`
+					goto constructive
+				`	"I witnessed the test of two wills."`
+					goto willed
+				`	"The meeting was poignant."`
+					goto poignant
+				`	"Very aggressive and confronting."`
+			action
+				set haiignorance
+			`	You hear many sighs, and several Elders shrug, but nobody seems surprised, just disappointed. "They grow ever more brazen in their desire for change," says the Elder. "Thank you, Captain <last>."`
+				goto next
+			label willed
+			action
+				set haiignorance
+			`	Many elders chuckle around the room. The Elder says, "Ah, our pair of old rivals, divided by Cloudfire. The embers are not yet quenched then." At this, Teeneep rolls her eyes and the Elder departs.`
+				goto next
+			label poignant
+			action
+				set haiinsight
+			`	There is a quiet shuffling as some of the elders sit up a little straighter. The Elder says, "A common ground between rivals, a fearsome thing indeed."`
+				goto next
+			label constructive
+			action
+				set haiinsight
+			`	A roomful of laughter engulfs your words, but many Elders appear offended by their peers' mockery. The Elder who questioned you explains, "The Elders and the Unfettered have different views of what constitutes a 'constructive discussion.' You lack the biases of a Hai. If you say it was constructive, I believe you." The tone of the room changes after that, as the laughter ends. Most Elders appear to agree with your assessment, and many who laughed look ashamed.`
+			label next
+			`	Sayari gives you a meaningful look and whispers, "A lot of them paid attention to that..."`
+			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are, all of you, cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`
```

Comment:
Teeneep is the threat, she is a threat to the established order.

Reply:
Could it be made clearer then? I never would have guessed thats what was meant here

Reply:
You could tell the player that Gemnali points her finger at Teeneep before she says "This threat cannot stand", and after "Order through security". There might be another way, because I really like that line building up to the end and putting that there might make it a bit worse.

Reply:
I mean, technically it's the threat that Teeneep *represents*: a person willing to wield the Hai's internal disunity against them to coerce the Elder's into a course of action they did not themselves find acceptable.
Teeneep is the threat as shorthand for that though.

I had thought the "this one here" called out teeneep clearly enough, but I guess not.

Reply:
`This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"` I don't see a this one here in here?
mb it was added since?

Reply:
It's earlier in the diatribe, two more sentences back, part of the framing statements.

Reply:
~~stop linking phrases so far way~~